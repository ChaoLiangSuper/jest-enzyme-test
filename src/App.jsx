import React from 'react';
import Add from './Add';
import Bigger from './Bigger';

class App extends React.Component {
  render() {
    const a = 1;
    const b = 3;
    const props = { a, b };
    return (
      <React.Fragment>
        <Add {...props} />
        <Bigger {...props} />
      </React.Fragment>
    );
  }
}

export default App;
