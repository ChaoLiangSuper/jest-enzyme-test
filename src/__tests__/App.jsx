import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

it('should render default component', () => {
  const component = renderer.create(<App />);
  const snap = component.toJSON();
  expect(snap).toMatchSnapshot();
});
