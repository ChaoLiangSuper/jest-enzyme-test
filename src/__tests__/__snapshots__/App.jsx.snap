// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render default component 1`] = `
Array [
  <div>
    add: 
    4
  </div>,
  <div>
    getBiggerNum: 
    3
  </div>,
]
`;
